pkg/v1/resource/certconfig/create_test.go

```diff
@@ -196,12 +196,12 @@ func Test_newCreateChange(t *testing.T) {
 			},
 			currentState: []*v1alpha1.CertConfig{
 				newCertConfig("cluster-1", certs.APICert),
-				newCertConfig("cluster-1", certs.CalicoCert),
+				newCertConfig("cluster-1", certs.Cert("calico")),
```

Comment:
As in PR description all versions prior to v8 have `calico` cert retained. I used this hack as it's removed from `certs` library.